src/main/java/org/betonquest/betonquest/modules/config/Patcher.java

```diff
@@ -85,6 +86,7 @@ public class Patcher {
      * @param config      the config that must be patched
      * @param patchConfig the patchConfig that contains patches
      */
+    @SuppressFBWarnings(value = "NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE", justification = "False Positive, default value is explicitly set.")
```

Comment:
The value of
`final String configVersion = config.getString(CONFIG_VERSION_PATH, TECHNICAL_DEFAULT_VERSION);`
can actually be null, depending on the implementation, there could be an explicit set null on this path, and depending on the implementation, this could also be returned. As we also have custom implementations of ConfigurationSection, i would ensure that this is not null instead of suppressing it

Reply:
The API explicitly defines that null cannot be returned when the second argument is not null via `@Contract("_, !null -> !null")` .
https://hub.spigotmc.org/javadocs/bukkit/org/bukkit/configuration/ConfigurationSection.html#getString(java.lang.String,java.lang.String)
This is a false positive.

Reply:
ok i see